graphql/resolvers.js

```diff
@@ -0,0 +1,130 @@
+require("dotenv").config();
+const { gql, UserInputError } = require("apollo-server");
+const jwt = require("jsonwebtoken");
+
+const operations = require("../data/operations");
+const users = require("../data/users");
+
+const resolvers = {
+  Query: {
+    getAccountAndMovements: () => operations,
+    me: (root, args, context) => {
+      return context.currentUser;
+    },
+  },
+  Mutation: {
+    addDeposit: (_, args) => {
+      const { number, type, amount } = args;
+      const findNumberAccount = operations
+        .map((operation) => operation.number)
+        .join();
+      const getActualDeposit = parseInt(
+        operations.map((operation) => operation.deposit)
+      );
+      if (findNumberAccount !== number) {
+        throw new UserInputError("Account number does not match", {
+          invalidArgs: number,
+        });
+      }
+      if (type !== "DEPOSIT") {
+        throw new UserInputError("Only deposit type operations are allowed", {
+          invalidArgs: type,
+        });
+      }
+      if (Math.sign(amount) === -1) {
+        throw new UserInputError(
+          "Negative numbers are not allowed in deposit type operations",
+          {
+            invalidArgs: amount,
+          }
+        );
+      }
+      const newBalance =
+        parseInt(operations.map((operation) => operation.balance)) + amount;
```

Comment:
Why would yo need this `parseInt`?

Reply:
To avoid returning a string. 
For instance, "8000" + 20, will return "80020" beceuse of type coercion.

Reply:
Yeah, that much I understand. My problem is the whole operations mapping 🤔 why parseInt an array?